src/Spreadsheet/DeliverySpreadsheetParser.php

```diff
@@ -168,10 +207,22 @@ private function applyTags(TaggableInterface $task, $tagsAsString)
         if (!empty($tagsAsString)) {
             $slugs = explode(' ', $tagsAsString);
             $tags = array_map([$this->slugify, 'slugify'], $slugs);
-            $task->setTags($tags);
+            $task->addTags($tags);
         }
     }
 
+    private function applyMetadata(Task $task, string $prefix, array $record)
+    {
+        $metadataColumn = $this->getColumn($prefix, 'metadata');
+
+        if (isset($record[$metadataColumn]) && !empty($record[$metadataColumn])) {
+            /* @throws JsonException */
+            $metadata = json_decode($record[$metadataColumn], JSON_THROW_ON_ERROR);
```

Comment:
I wouldn't use JSON format to send metadata, but another "simpler" format like for packages (`foo=bar baz=bat`), or even URL encoded (`foo=bar&baz=bat`).